Tests/MCPTests/Helpers/MockTransport.swift

```diff
@@ -1,4 +1,8 @@
-import Foundation
+import class Foundation.JSONEncoder
+import class Foundation.JSONDecoder
+import struct Foundation.Data
+import struct Foundation.POSIXError
```

Comment:
The reason why someone might need to fully-qualify `MCP.Method` or `MCP.Notification` is to fix a _different_ namespace collision with Foundation. 

As a workaround, you can [import individual declarations](https://nshipster.com/import/#importing-individual-declarations) as needed. Apologies in advance for any inconvenience this causes 🙇